Add `o!tFAC #1` contest article (#6964)
ppy/osu-wiki: wiki/Contests/o!tFAC/1/en.md

---
tags:

>   - o!tfac
>   - o!tfac#1

Reviewer: ```suggestion
  - o!tfac
  - o!tfac1
  - o!tfac 1
  - o!tfac#1
  - tfac
  - tfac1
  - tfac 1
  - tfac#1
```

---

# osu!taiko Featured Artist Cup #1

![o!tfac logo](img/logo.png)

> The **osu!taiko Featured Artist Cup #1** (***o!tfac #1***) is an osu!taiko mapping contest where contestants are simply asked to map a song from osu!'s Featured Artists.

Reviewer: ```suggestion
The **osu!taiko Featured Artist Cup #1** (***o!tfac #1***) is an osu!taiko mapping contest where contestants are simply asked to map a song from one of osu!'s Featured Artists.
```

> ## Special limitations
> 
> This iteration has none.
> 

Reviewer: i see

```suggestion
```

## Contest schedule

> | Event | Timestamp (UTC+00) |
> | --: | :-- |
> | Registration phase | Feb 12th, 2022 - Feb 27th, 2022 00:00 |
> | Submission phase | Feb 27th, 2022 - Mar 6th, 2022 00:00 |
> | Judging phase | Mar 6th, 2022 - Mar 20th, 2022 |
> | Results announcement | TBA |

Reviewer: ```suggestion
| Event | Timestamp (UTC) |
| --: | :-- |
| Registration phase | 2022-02-12/2022-02-27 00:00 |
| Submission phase | 2022-02-27/2022-03-06 00:00 |
| Judging phase | 2022-03-06/2022-03-20 |
| Results announcement | *TBA* |
```

## Organisation

> The osu!taiko Featured Artist Cup #1 is run by members of the following team:

Reviewer: ```suggestion
The osu!taiko Featured Artist Cup #1 is run by various community members.
```

| Host | ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) |
| Designer | ![][flag_MY] [\[Zeth\]](https://osu.ppy.sh/users/9912966) |
| Judge | ![][flag_US] [Alchyr](https://osu.ppy.sh/users/4993032), ![][flag_AR] [Axer](https://osu.ppy.sh/users/7299864), ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097), ![][flag_MY] [\[Zeth\]](https://osu.ppy.sh/users/9912966) |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter, Ranked status¹,  Unique profile badge (unconfirmed) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 3 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter |

![o!tfac #1 badge](img/badge.png)

¹: ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) and ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097) will mod and push the winning entry to ranked if the winner chooses to, and **if they get the necessary difficulties to form a rankable spread if needed**.

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1520800)
- [Discord server](https://discord.gg/jMsVqwPVrD)

> ## Podium
> 
> This contest is still ongoing and is currently in its judging phase.
> 
> | Place | User |
> | :-: | :-- |
> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | TBD |
> | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | TBD |
> | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | TBD |
> 

Reviewer: we usually just leave out provisional sections

```suggestion
```

## Ruleset

### Contest rules

> - **The contest will consist of one round where contestants will submit ONE difficulty of any licensed song they choose from the [Featured Artists](https://osu.ppy.sh/beatmaps/artists) list.**

Reviewer: ```suggestion
- **Contestants will submit ONE difficulty of any licensed song they choose from the [Featured Artists](https://osu.ppy.sh/beatmaps/artists) library.**
```

- **Your map must be an osu!taiko map.**

> - **All submissions must be rankable.** No mapset submitted must be intentionally unrankable. Mistakes happen and points will be deducted for them, but this is a regular mapping contest, not aspire. (You're allowed to ignore spread rules for the sake of this contest)

Reviewer: ```suggestion
- **All submissions must be rankable.** No beatmap submitted must be intentionally unrankable. Mistakes happen and points will be deducted for them, but this is a regular mapping contest, not aspire. (You're allowed to ignore spread rules for the sake of this contest.)
```

> - **Maps must NOT be uploaded / shared with others before judging and results are announced.** Sharing your entry publicly by any means before the results are announced will get your entry disqualified, no questions asked.

Reviewer: ```suggestion
- **Maps must NOT be uploaded/shared with others before judging and results are announced.** Sharing your entry publicly by any means before the results are announced will get your entry disqualified, no questions asked.
```

- **Submissions will not be accepted after the deadlines.** No exceptions.

### Judging criteria

> - **Judge's Impression (5 Points):** Points will be assigned based on what the judge thinks about your map.
> 
> - **Ranking Criteria (5 Points):** Points will be assigned based on how well the map abides by the Ranking Criteria.
> 
> - **Structure (10 Points):** Points will be assigned based on how well the map is internally designed (consistency, concept introduction, etc.).
> 
> - **Relevance (10 Points):** Points will be assigned based on how well the map represents the song and how well it plays.
> 
> - **Creativity (10 Points):** Points will be assigned based on how the submission uses creative and fitting elements to make itself stand out.

Reviewer: ```suggestion
- **Judge's Impression (5 Points):** Points will be assigned based on what the judge thinks about your map.
- **Ranking Criteria (5 Points):** Points will be assigned based on how well the map abides by the Ranking Criteria.
- **Structure (10 Points):** Points will be assigned based on how well the map is internally designed (consistency, concept introduction, etc.).
- **Relevance (10 Points):** Points will be assigned based on how well the map represents the song and how well it plays.
- **Creativity (10 Points):** Points will be assigned based on how the submission uses creative and fitting elements to make itself stand out.
```
